### PR TITLE
Patch email notifier episode parsing. Fixes #4159

### LINKED ISSUE #4159: Parsing in Notification Emails
**Branch/Commit:** 77b20916ab577b82112ebc624f816054788c63f2 / master
**OS:** Server 2012 R2
**What you did:** - Sent a notification email
**What happened:** Email body parsing incorrect
**What you expected:** Parsing to be correct 
**Logs:** NA - See screenshots below! 

Hey guys, my TV show notification emails are showing like below, they display the same on webmail / outlook / android mail app etc.

image

image
image
image
image

### PR DESCRIPTION
**Pull request by sharkykh**
Fixes #4159

Some of my regex tests: https://regex101.com/r/KxDpoT/2

#3062 would be the better option, but this should be good for now.

### CODE REVIEW

File: medusa/notifiers/emailnotify.py [hunk 1]
```diff
@@ -20,6 +20,21 @@
 
 
 class Notifier(object):
+
+    # Possible patterns:
```
Comment (p0psicles) [inline medusa/notifiers/emailnotify.py]: Use docstring for this?

File: medusa/notifiers/emailnotify.py [hunk 2]
```diff
@@ -325,12 +357,25 @@ def _sendmail(self, host, port, smtp_from, use_tls, user, pwd, to, msg, smtpDebu
             self.last_err = '{0}'.format(error)
             return False
 
-    @staticmethod
-    def _parseEp(ep_name):
+    @classmethod
+    def _parseName(cls, ep_name):
```
[the remark this answers is not in the document]
Comment (sharkykh) [reply medusa/notifiers/emailnotify.py]: Yeah already did that :)

File: medusa/notifiers/emailnotify.py (hunk 1 above)
Comment (sharkykh) [reply medusa/notifiers/emailnotify.py]: But it doesn't describe the notifier class, just the possible patterns for the regex pattern below.

Comment (p0psicles) [reply medusa/notifiers/emailnotify.py]: Then we make a short class docstring?

File: medusa/notifiers/emailnotify.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (p0psicles) [reply medusa/notifiers/emailnotify.py]: Yeah old comment.

**Pull request merged**